src/backend/sgx/thread.rs

```diff
@@ -75,26 +71,39 @@ impl super::super::Keep for super::Keep {
 fn sgx_enarxcall<'a>(enarxcall: &'a mut Payload, data: &'a mut [u8]) -> Result<Option<Item<'a>>> {
     match enarxcall {
         item::Enarxcall {
-            num,
+            num: item::enarxcall::Number::Cpuid,
```

Comment:
Same here for bringing Enarxcall and Number into scope.

Reply:
I think they right way to do this would be to first split each call kind handling into a separate file where we could bring these in scope indeed.
Referring to these via `Number::X` can be too ambiguous here, but if we had `enarxcall.rs`, `gbdcall.rs` etc. that would be better organized